dvc/command/commit.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import unicode_literals
+
+import dvc.logger as logger
+from dvc.exceptions import DvcException
+from dvc.command.base import CmdBase
+
+
+class CmdCommit(CmdBase):
+    def run(self):
+        if not self.args.targets:
+            self.args.targets = [None]
+
+        for target in self.args.targets:
+            try:
+                self.repo.commit(
+                    target,
+                    with_deps=self.args.with_deps,
+                    recursive=self.args.recursive,
+                    force=self.args.force,
+                )
+            except DvcException:
+                logger.error(
+                    "failed to commit{}".format(
+                        (" " + target) if target else ""
+                    )
+                )
+                return 1
+        return 0
+
+
+def add_parser(subparsers, parent_parser):
+    COMMIT_HELP = "Record changes to the repository."
+    commit_parser = subparsers.add_parser(
+        "commit",
```

Comment:
What is a revert operation for `dvc commit file`?

Reply:
@dmpetrov there is no anti-commit currently. By logic it is something that would remove cache for that specific dvc file, but, as we know, it is not possible to do, since that same cache file might be used elsewhere, so we can't just remove it without analyzing everything else like gc does. So something like `dvc gc -aT stage.dvc` would be an anti-commit.